AuthishModule/ValidationService.cs

```diff
@@ -7,6 +9,7 @@ static class ValidationService
     {
         private static readonly ILog Log = LogManager.GetLogger(typeof(AuthishHandler));
         private static readonly string AuthishPassword = ConfigurationManager.AppSettings["AuthishPassword"];
+        private static readonly string[] AuthishWhitelistedPaths = (ConfigurationManager.AppSettings["AuthishWhitelistedPaths"] ?? "").Split(',');
```

Comment:
Right, sorry, my fault (not on top of the PR workflow after too much PowerPoint work lately ;-))